Update upgrade-dbt-version-in-cloud.md (#6748)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

1. Click your account name from the left side panel and select **Account settings**. 
2. Choose **Credentials** from the sidebar and select a project. This opens a side panel.
3. In the side panel, click **Edit** and scroll to the **User development settings** section. 
4. Choose a version from the **dbt version** dropdown and click **Save**.

  An example of overriding the configured version to ["Latest" release track](/docs/dbt-versions/cloud-release-tracks) for the selected project:

  <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-override-version.png" width="60%" title="Example of overriding the dbt version on your user account"/>

5. (Optional) Verify that dbt Cloud will use your override setting to build the project by invoking a `dbt build` command in the dbt Cloud IDE's command bar. Expand the **System Logs** section and find the output's first line. It should begin with `Running with dbt=` and list the version dbt Cloud is using. <br /><br />

>    For users on Release tracks, the output will display `Running dbt...` instead of a specific version like `Running dbt=1.9.X`, reflecting the flexibility and continuous updates provided by the release track functionality.

Author: ```suggestion
   For users on Release tracks, the output will display `Running dbt...` instead of a specific version, reflecting the flexibility and continuous automatic updates provided by the release track functionality.
```

Configure your project to use a different dbt Core version than what's configured in your [development environment](/docs/dbt-cloud-environments#types-of-environments). This _override_ only affects your user account, no one else's. Use this to safely test new dbt features before upgrading the dbt version for your projects. 

1. Click your account name from the left side panel and select **Account settings**. 
2. Choose **Credentials** from the sidebar and select a project. This opens a side panel.

> 3. In the side panel, click **Edit** and scroll to the **User development settings** section. Choose a version from the **dbt version** dropdown and click **Save**.

Reviewer: Hiya @mirnawong1 

This is a non-blocker. Would it be worth using a versionless image (to silently encourage people to upgrade to versionless)? I can grab one for you if you like?

Kind Regards
Natalie

Author: was this something you were going to address in your ui project? i noticed the [images](https://docs.getdbt.com/docs/dbt-versions/upgrade-dbt-version-in-cloud) were old but wasn’t sure if you were addressing this already. if not, i can add a screenshot even though its out of scope of the pr

#### Testing your changes before upgrading
Once you know what code changes you'll need to make, you can start implementing them. We recommend you:

> - Create a separate dbt project, **Upgrade project**, to test your changes before making them live in your main dbt project.

Reviewer: Hiya @mirnawong1 

Is the **Upgrade project** a step (ie do users have to navigate somewhere in their instance to get to this)?

Kind Regards
Natalie

Author: i'm not sure! it was there already and i was just adding bullets. i think it's a project name - so happy to change it. good catch!

  - If upgrading directly to the latest version results in too many issues, try testing your project iteratively on successive minor versions. There are years of development and a few breaking changes between distant versions of dbt Core (for example, 0.14 --> 1.0). The likelihood of experiencing problems upgrading between successive minor versions is much lower, which is why upgrading regularly is recommended.
- Once you have your project compiling and running on the latest version of dbt in the development environment for your `dbt-version-upgrade` branch, try replicating one of your production jobs to run off your branch's code.
- You can do this by creating a new deployment environment for testing, setting the custom branch to 'ON' and referencing your `dbt-version-upgrade` branch. You'll also need to set the dbt version in this environment to the latest dbt Core version.

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions/upgrade-environment.png" width="80%" title="Setting your testing environment" />

- Then add a job to the new testing environment that replicates one of the production jobs your team relies on.

>   - If that job runs smoothly, you should be all set to merge your branch into main and, change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.

Author: @nataliefiann , i turned it into two separate bullets since. i wasn't sure if the `and,` was a right fit.

````suggestion
  - If that job runs smoothly, you should be all set to merge your branch into main. 
  - Then change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.
```

>   - If that job runs smoothly, you should be all set to merge your branch into main and change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.

Reviewer: ```suggestion
  - If that job runs smoothly, you should be all set to merge your branch into main and, change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.
```

Reviewer: Hiya @mirnawong1 

I've added this comma to break the sentence up a bit

Kind Regards
Natalie

Author: thanks @nataliefiann , not sure if the command should go before the 'and'. but i turned it into two separate bullets
